addons/project/models/project_task.py

```diff
@@ -970,6 +971,10 @@ def create(self, vals_list):
                     user_ids = self._fields['user_ids'].convert_to_cache(vals.get('user_ids', []), self)
                     if self.env.user.id not in list(user_ids) + [SUPERUSER_ID]:
                         vals['user_ids'] = [Command.set(list(user_ids) + [self.env.user.id])]
+            # Ensure that when a task is created from the web editor, the user_ids is not filled with the odoo bot.
+            elif self.env.context.get('website_id', False):
+                vals['user_ids'] = False
+
```

Comment:
Not our area of concern but this seems like it might be too broad. Shouldn't you either fix it in a website form override (tricky I guess), or check explicitly for odoobot?